src/components/grid/tile/Tile.tsx

```diff
@@ -2,51 +2,43 @@ import * as React from 'react'
 import styled, { css } from 'styled-components'
 
 import { squareSize } from '../constants'
-import { TerrainType } from '../../terrains/base/TerrainType'
-import { TerrainModifierType } from '../../terrains/modifiers/TerrainModifierType'
-import { Unit } from '../../units/Unit'
+import { Unit } from '../../units/units'
+import { TerrainType } from 'src/components/terrains/base/terrains'
+import { TerrainModifierType } from 'src/components/terrains/modifiers/terrainModifiers'
 
-const StyledTile = styled.span`
-  height: ${squareSize.height};
-  width: ${squareSize.width};
-  display: inline-block;
-  border: 1px solid black;
-  box-sizing: border-box;
-  ${props =>
-      props['data-terrain'] === 'SEA' &&
-      css`
-          background: Aqua;
-      `}
-  ${props =>
-      props['data-terrain'] === 'SNOW' &&
-      css`
-          background: White;
-      `}
-  ${props =>
-      props['data-terrain'] === 'DESERT' &&
-      css`
-          background: Yellow;
-      `}
-  ${props =>
-      props['data-terrain'] === 'DIRT' &&
-      css`
-          background: Olive;
-      `}
-  ${props =>
-      props['data-terrain'] === 'GRASS' &&
-      css`
-          background: Green;
-      `}  
+const getColor = (terrain: TerrainType) => {
```

Comment:
this should change in the future, I'd rather name it in a generic way like 'getStyle'